config/manager/deployment.yaml

```diff
@@ -25,6 +25,7 @@ spec:
         securityContext:
           allowPrivilegeEscalation: false
           readOnlyRootFilesystem: true
+          fsGroup: 1337
```

Comment:
You need to set this at the pod level otherwise it won't work. See info [here](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container):
> Container settings do not affect the Pod's Volumes.